README.adoc

```diff
@@ -85,6 +85,25 @@ oc process \
 | oc apply -n poolboy -f -
 ----
 
+### Using Helm
+
+Helm v3 support only.
+
+. Clone this repo to your local workstation and change directory:
++
+----
+git clone https://github.com/redhat-cop/poolboy.git
+cd poolboy
+----
+
+. Install with Helm
++
+----
+helm template helm | oc apply -f -
+----
+
+CAUTION: The anarchy namespace must already exist as we will create objects in that namespace. The default is `anarchy-operator` but can be configured in `value.yaml` or with `--set anarchy.namespace=<anarchy namespace>` above.
```

Comment:
Poolboy is not specifically tied to Anarchy... I see the anarchy-integration below... I'm not sure where exactly the Anarchy + Poolboy stuff belongs. Perhaps it actually belongs in the babylon repository rather than either Anarchy or Poolboy? That's the direction I am leaning, that it was a mistake to put this integration in this repository to begin with. Can we remove the Anarchy integration from this PR and introduce it into a separate PR against redhat-cop/babylon?